src/data/Graph.js

```diff
@@ -182,6 +179,8 @@ graphProto.addEdge = function (n1, n2, dataIndex) {
     this.edges.push(edge);
     edgesMap[key] = edge;
 
+    createEdgeMapForCurveness(n1, n2, seriesModel, dataIndex);
```

Comment:
I think `createEdgeMapForCurveness` should better not be added in `Graph addEdge`.
Because `Graph` is not only used in graph series but also used in other series like sankey.
And `Graph` it seams that do not need to depends on the module `multipleGraphEdgeHelper`,
and do not necessary to introduce an param of `seriesModel` here.

The calling of `createEdgeMapForCurveness` can be done in `GraphSeries getInitialData`.
See the comment in `GraphSeries getInitialData`.